### PR TITLE
Fix part of koppor#475 - remove some illegal tags in javadoc to pass the javadoc check

### PR DESCRIPTION
**Pull request by desperadomzc**
Fixes https://github.com/koppor/jabref/issues/475
remove some illegal tags in javadoc and passes the javadoc check

### Mandatory checks

- [ ] Change in CHANGELOG.md described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [ ] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [x] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [x] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at https://github.com/JabRef/user-documentation/issues or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

**Comment by koppor** [comment]
Follow-up of https://github.com/JabRef/jabref/pull/9984

**Comment by desperadomzc** [comment]
Hi @koppor, it failed the Fetcher Tests, but the changes are only in comments. I have met this problem before, but it passed the test the second time.

**Comment by koppor** [comment]
@desperadomzc Why did you close this PR? It was close to be ready to be merged!

File: src/main/java/org/jabref/gui/autocompleter/AutoCompletionTextInputBinding.java [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /**
  * Copyright (c) 2014, 2015, ControlsFX
  * All rights reserved.
- *
+ * <p>
```
[the remark this answers is not in the document]
Comment (desperadomzc) [reply src/main/java/org/jabref/gui/autocompleter/AutoCompletionTextInputBinding.java:4]: I'll handle it.

[the remark this answers is not in the document]
Comment (desperadomzc) [reply src/main/java/org/jabref/gui/autocompleter/AutoCompletionTextInputBinding.java]: Hi @koppor ,I have pushed the changes to the branch ```SPI-2023:fix-issue-475```. Sorry for the delay.

File: src/main/java/org/jabref/gui/autocompleter/AutoCompletionTextInputBinding.java [hunk 2]
```diff
@@ -1,7 +1,6 @@
 /**
  * Copyright (c) 2014, 2015, ControlsFX
  * All rights reserved.
- *
```
Comment (koppor) [inline src/main/java/org/jabref/gui/autocompleter/AutoCompletionTextInputBinding.java:4]: These lines are now deleted... I will re-add them to keep things moving.

**Comment by koppor** [comment]
Maintainers were not given access to modify files of this PR. Thus, I cannot do anything. Could you please revert your changes? If you are at the branch `fix-issue-475`, it is just following call:

git checkout main -- src/main/java/org/jabref/gui/autocompleter/AutoCompletionTextInputBinding.java

Then commit and push ^^

**Review by koppor — APPROVED** [review]

**Pull request merged**